definitions/types/gcp-grn.json

```diff
@@ -4,11 +4,12 @@
   "title": "GCP Resource Name (GRN)",
   "description": "GCP Resource Name (GRN)",
   "examples": [
-    "projects/<my-project>/locations/<location>/instances/<my-instance>",
-    "projects/<my-project>/locations/<location>/clusters/<my-cluster>",
-    "projects/<my-project>/zones/<zone>/instances/<node-pool-id>"
+    "projects/my-project/global/networks/my-network",
```

Comment:
The regex didn't support this, it expected another slash and more characters. I chopped off that last bit `\/` and tested these in the online tool.